server/app.js

```diff
@@ -1,82 +1,89 @@
-const colors = require("colors");
-const path = require("path");
-const http = require("http");
-const express = require("express");
-const socketio = require("socket.io");
-const { notFound, errorHandler } = require("./middleware/error");
-const connectDB = require("./db");
-const { join } = require("path");
-const cookieParser = require("cookie-parser");
-const logger = require("morgan");
+const colors = require('colors');
+const path = require('path');
+const http = require('http');
+const express = require('express');
+const socketio = require('socket.io');
+const { notFound, errorHandler } = require('./middleware/error');
+const connectDB = require('./db');
+const { join } = require('path');
+const cookieParser = require('cookie-parser');
+const logger = require('morgan');
 const passport = require('passport');
 const session = require('express-session');
 require('./utils/oauthGoogleStrategy');
 
-const authRouter = require("./routes/auth");
-const userRouter = require("./routes/user");
-const oauthRouter = require("./routes/oauth");
-const appointmentRouter = require("./routes/appointment");
-const meetingRouter = require("./routes/meeting");
-
+const authRouter = require('./routes/auth');
+const userRouter = require('./routes/user');
+const oauthRouter = require('./routes/oauth');
+const appointmentRouter = require('./routes/appointment');
+const meetingRouter = require('./routes/meeting');
 
 const { json, urlencoded } = express;
 
 connectDB();
 const app = express();
 const server = http.createServer(app);
 
+// Initializes our cloudinary
+cloudinary.config({
+	cloud_name: 'calend-app',
+	api_key: process.env.CLOUDINARY_API_KEY,
+	api_secret: process.env.CLOUDINARY_API_SECRET,
+	secure: true
+});
+
 const io = socketio(server, {
-  cors: {
-    origin: "*",
-  },
+	cors: {
+		origin: '*'
+	}
 });
 
-io.on("connection", (socket) => {
-  console.log("connected");
+io.on('connection', (socket) => {
+	console.log('connected');
 });
 
-if (process.env.NODE_ENV === "development") {
-  app.use(logger("dev"));
+if (process.env.NODE_ENV === 'development') {
+	app.use(logger('dev'));
 }
 app.use(json());
 app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(session({ secret: process.env.SESSION_SECRET }));
 app.use(passport.initialize());
 app.use(passport.session());
-app.use(express.static(join(__dirname, "public")));
+app.use(express.static(join(__dirname, 'public')));
 
 app.use((req, res, next) => {
-  req.io = io;
-  next();
+	req.io = io;
```

Comment:
here

Reply:
Hi, do you have any clues as to how to fix this? On my client it shows that I have 2 spaces but when I push it gets messed up.